.github/workflows/ui-tests/cypress/integration/create_spec.js

```diff
@@ -28,6 +28,53 @@ describe('User can navigate to create account screen', function() {
     })
 })
 
+describe('Signup is unsuccessful with long username', function() {
+    const invalidFeedback = Cypress.env('messages').invalidFeedback
```

Comment:
since this is used in multiple describe blocks, it makes sense to move to import at beginning of file
example: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/5ae1f311252d0bf85a1cd65656fb84485cbb8952/.github/workflows/ui-tests/cypress/integration/transfer_spec.js#L24